templates/homepage.html

```diff
@@ -21,11 +21,13 @@ <h4>What is this Gamification Website About?</h4>
                     tokens. When you have collected enough tokens, you can trade them in for marks in the course! That
                     means, the more you practice here, the more you are able to make up for lost marks on assignments
                     and exams (based on pre-set rules).
-
+                    <br>
```

Comment:
Can you please seperate paragraphs by <p> tag rather than using <br> ?

Reply:
Can do